compiler/damlc/tests/src/DA/Test/DamlRenamer.hs

```diff
@@ -77,6 +77,7 @@ runDamlRename damlc damlFile = do
       , "--ghc-option=-dumpdir=" <> tempDir
       , "--ghc-option=-ddump-file-prefix=o."
       , "--ghc-option=-ddump-rn"
+      , "--ghc-option=-dsuppress-uniques"
```

Comment:
In https://github.com/digital-asset/daml/pull/16379 I missed that the output with `-dumpdir` includes uniques while the previous approach that extracted the renamer output from stderr didn't have them. Thankfully I found this flag which gets us closer to the old behavior. This ensures that this test won't need updating if the unique supply changes (e.g. when adding new functions in `daml-prim` or `daml-stdlib`)

Reply:
we're now not testing the behaviour of the uniques at all here, is that okay? do we have tests elsewhere that ensure that part is behaving correctly?

Reply:
to be clear this wasn't testing the uniques prior to #16379 either. I don't think we test that part directly but it's inherited from GHC which does test it

Reply:
Ah right okay